model/survey-instance.js

```diff
@@ -21,8 +21,33 @@ const Sequelize = require('sequelize');
 module.exports = function (sequelize) {
     sequelize.define('survey_instance',
         {
-            name: {
-                type: Sequelize.STRING
+            startTime: {
+                type: Sequelize.DATE,
+                allowNull: false,
+                validate: {
+                    notEmpty: true
+                }
+            },
+            endTime: {
+                type: Sequelize.DATE,
+                allowNull: false,
+                validate: {
+                    notEmpty: true
+                }
+            },
+            userSubmissionTime: {
+                type: Sequelize.DATE
+            },
+            actualSubmissionTime: {
+                type: Sequelize.DATE
+            },
+            surveyInstanceCompleted: {
+                type: Sequelize.BOOLEAN,
+                allowNull: false,
+                defaultValue: false,
+                validate: {
+                    notEmpty: true
```

Comment:
I removed the validator. Will that suffice? I thought of a default value as false to be present for this boolean. Hence i should get rid of allowNull right?